virt_lightning/virt_lightning.py

```diff
@@ -110,7 +110,6 @@ def configure_domain(self, domain, user_config):
         config = {
             "groups": [],
             "memory": 768,
-            "python_interpreter": "/usr/bin/python3",
```

Comment:
Will it crash here https://github.com/virt-lightning/virt-lightning/blob/61fa6ef9cd2cce6a70a2b873e20d0d81a84edccd/virt_lightning/virt_lightning.py#L128 ?

Reply:
Rah, good point. Thanks. I close the PR for now.